indico/htdocs/js/indico/jquery/clearableinput.js

```diff
@@ -31,15 +31,16 @@
         _create: function() {
             var self = this;
 
-            self.buttonBox = $('<span class="button-box"></span>');
-            self.clearIcon = $('<a class="input-clear icon-close"></a>')
+            self.buttonBox = $('<span>').addClass('button-box');
```

Comment:
Personally I prefer the syntax
``` js
self.buttonBox = $('<span>', {
    class: 'button-box'
});
```
which is also proven to be a bit faster. But in this case, your way is more readable, so I don't have a strong opinion.